src/lib.rs

```diff
@@ -130,83 +144,166 @@ impl Timecode {
   }
 }
 
+impl Timecode {
+  pub fn hours(&self) -> u8 {
+    self.hours
+  }
+
+  pub fn minutes(&self) -> u8 {
+    self.minutes
+  }
+
+  pub fn seconds(&self) -> u8 {
+    self.seconds
+  }
+
+  pub fn fraction(&self) -> &Fraction {
+    &self.fraction
+  }
+}
+
+impl ToString for Timecode {
+  fn to_string(&self) -> String {
+    let fraction = self.fraction.to_string();
+
+    format!(
+      "{:02}:{:02}:{:02}{}",
+      self.hours, self.minutes, self.seconds, fraction
+    )
+  }
+}
+
+impl From<&Timecode> for f64 {
+  fn from(timecode: &Timecode) -> Self {
+    (timecode.hours as f64) * 3600.0 + (timecode.minutes as f64) * 60.0 + (timecode.seconds as f64)
+  }
+}
+
```

Comment:
Move unitary tests in a test module
(See https://doc.rust-lang.org/book/ch11-01-writing-tests.html)